app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        try:
+            vaccine_info = visitor["vaccine"]
+        except KeyError:
+            raise NotVaccinatedError
```

Comment:
The `NotVaccinatedError` is raised without a descriptive message. According to the checklist, you should provide a descriptive message when raising exceptions. Consider using `raise NotVaccinatedError('Visitor is not vaccinated')`.